views/index.tmpl

```diff
@@ -150,7 +145,9 @@
 <script>
 var youtubeAPIKey = {{.PageContext.Youtube.APIKey}};
 var youtubePlaylistID = "{{.PageContext.Youtube.PlaylistID}}";
+var myRadioAPIKey = {{.PageContext.MyRadioAPIKey}};
```

Comment:
ideally the js for the message box would be in the message_box.tmpl but I can't access .PageContext inside that. Any ideas @LordAro @MattWindsor91

Reply:
Not that this helps much, but it would seem that the go linter is complaining about these lines a fair bit.

Reply:
Go linter? But this is html/js

Reply:
Probably a miss trigger, I think it's trying to parse bits of that for some reason. It's obviously actually correct :)